src/generators/llms-txt/index.mjs

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+import { readFile, writeFile } from 'node:fs/promises';
+import { join } from 'node:path';
+import { generateDocEntry } from './utils/generateDocEntry.mjs';
+import { LATEST_DOC_API_BASE_URL } from './constants.mjs';
+
+/**
+ * @typedef {Array<ApiDocMetadataEntry>} Input
+ *
+ * @type {GeneratorMetadata<Input, string>}
+ */
+export default {
+  name: 'llms-txt',
+  version: '1.0.0',
+  description: 'Generates a llms.txt file of the API docs',
+  dependsOn: 'ast',
+
+  /**
+   * @param {Input} input The API documentation metadata
+   * @param {Partial<GeneratorOptions>} options Generator options
+   * @returns {Promise<string>} The generated documentation text
+   */
+  async generate(input, options) {
+    const template = await readFile(
+      join(import.meta.dirname, 'template.txt'),
+      'utf-8'
+    );
+
+    const apiDocEntries = input.map(generateDocEntry).filter(Boolean);
+
+    const introductionEntries = [
+      `- [About this documentation](${LATEST_DOC_API_BASE_URL}/api/documentation.md)`,
```

Comment:
This seems hard-coded and likely to fail in the future; can we obtain this data from another source?